vladislavVVVVV/SOA
# Add recipes web api

Test plan:

- Install VS Code
- Install Docker
- Open webapi project
- Add next packages:
> - `dotnet add package Microsoft.EntityFrameworkCore.Design`
> - `dotnet add package microsoft.entityframeworkcore.sqlserver.design`
- run in console:
> - `docker run -d --name Soarh -e 'ACCEPT_EULA=Y' -e 'SA_PASSWORD=Hard_pass_123' -p 1433:1433 microsoft/mssql-server-linux`
- make sure that container is active, use `docker ps`
- run `dotnet watch run`
- see `http://localhost:5000/swagger/index.html`
![Снимок экрана 2021-04-12 в 03 18 27](https://user-images.githubusercontent.com/49831502/114326499-c1b35d80-9b3d-11eb-9ef4-0003bcc274bc.png)
- make sure that all api calls work as expected
